### ISSUE TITLE
Missing dependency when using though Maven

### ISSUE THREAD

**Issue by jmccrae** [ispras/atr4s#2]
It seems that the Maven POM does not declare a dependency to log4j version 2.4

```
[ERROR] error: missing or invalid dependency detected while loading class file 'TermCandidatesCollector.class'.
[INFO] Could not access term apache in package org,
[INFO] because it (or its dependencies) are missing. Check your build definition for
[INFO] missing or conflicting dependencies. (Re-run with `-Ylog-classpath` to see the problematic classpath.)
[INFO] A full rebuild may help if 'TermCandidatesCollector.class' was compiled against an incompatible version of org.
[ERROR] error: missing or invalid dependency detected while loading class file 'TermCandidatesCollector.class'.
[INFO] Could not access term logging in value org.apache,
[INFO] because it (or its dependencies) are missing. Check your build definition for
[INFO] missing or conflicting dependencies. (Re-run with `-Ylog-classpath` to see the problematic classpath.)
[INFO] A full rebuild may help if 'TermCandidatesCollector.class' was compiled against an incompatible version of org.apache.
[ERROR] error: missing or invalid dependency detected while loading class file 'FeatureConfig.class'.
[INFO] Could not access term apache in package org,
[INFO] because it (or its dependencies) are missing. Check your build definition for
[INFO] missing or conflicting dependencies. (Re-run with `-Ylog-classpath` to see the problematic classpath.)
[INFO] A full rebuild may help if 'FeatureConfig.class' was compiled against an incompatible version of org.
[ERROR] error: missing or invalid dependency detected while loading class file 'FeatureConfig.class'.
[INFO] Could not access term logging in value org.apache,
[INFO] because it (or its dependencies) are missing. Check your build definition for
[INFO] missing or conflicting dependencies. (Re-run with `-Ylog-classpath` to see the problematic classpath.)
[INFO] A full rebuild may help if 'FeatureConfig.class' was compiled against an incompatible version of org.apache.
```

Fixed by adding the following to my project's pom.xml

```
        <dependency>
            <groupId>org.apache.logging.log4j</groupId>
            <artifactId>log4j-core</artifactId>
            <version>2.4</version>
        </dependency>
```

**Comment by astrakhantsev** [comment]
Thank you!
Atr4s lib always contained apache.logging.log4j in dependencies, but gradle plugin that I used to upload atr4s to JCenter and mavenCentral created pom.xml with runtime scope for no reason, see: https://github.com/gradle/gradle/issues/1118
Now, in 1.1, it has correct scope, so you don't need to add this dependency to your project. 
I've checked it, but it would be good if you check it for your project.
